skillmap/src/components/InfoPanel.tsx

```diff
@@ -125,7 +125,7 @@ function mapStateToProps(state: SkillMapState, ownProps: any) {
         title: node?.displayName || state.title,
         subtitle,
         description: isActivity ? (node as MapActivity).description : state.description,
-        imageUrl: node?.imageUrl,
+        imageUrl: node ? node?.imageUrl : state.bannerImageUrl,
```

Comment:
```suggestion
        imageUrl: node?.imageUrl || state.bannerImageUrl,
```

Reply:
ah wait sorry i committed then uncommited, i think we actually want it to be null if a node is present but doesn't have an imageUrl (eg some certificate nodes, maybe?), it'll just be the default "no image" icon in that case